corehq/apps/users/bulkupload.py

```diff
@@ -363,6 +364,20 @@ def _set_progress(progress):
                 'username': raw_username(username) if username else None,
                 'row': row,
             }
+
+            is_active = row.get('is_active')
+            if type(is_active) in (unicode, str):
```

Comment:
(not blocking) A better way to do this might be `isinstance(is_active, basestring)` since that will catch subclasses of `str` and `unicode` as well.